src/connmgr/client.rs

```diff
@@ -420,46 +425,30 @@ impl Connections {
 
             assert!(count <= zhttppacket::IDS_MAX);
 
-            let zreq = zhttppacket::Request {
+            let zresp = zhttppacket::Response {
```

Comment:
This should be a response packet here rather than a request packet. We've simply been lucky that they serialize the same.